File: server/PredlaganjeSaradnjeIRCDemo/PredlaganjeSaradnjeIRC.Data/Model/Company.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace PredlaganjeSaradnjeIRC.Data.Model
{
    public class Company
    {
        [Required]
        public int Id { get; set; }
        public IEnumerable<Location> Locations { get; set; }
        public IEnumerable<Contact> Contacts  { get; set; }
        public string Name { get; set; }
        public string? Username { get; set; }
        public string? Password { get; set; }
    }
}

File: client/fpis-app/src/model/Company.ts
import {Location} from './Location'
import {Contact} from './Contact'
export class Company{
    id:number;

    locations:Location[];

    contacts:Contact[];

    name:string;

    username:string;

    password:string;

    constructor(id:number,locations:Location[],contacts:Contact[],name:string,username:string,password:string){
        this.id=id;
        this.locations=locations;
        this.contacts=contacts;
        this.name=name;
        this.username=username;
        this.password=password;
    }
}